doc/source/security/enabling-master-key.rst

```diff
@@ -0,0 +1,206 @@
+    | TABLE_SCHEMA         | TABLE_NAME        | CREATE_OPTIONS               |
+    +----------------------+-------------------+------------------------------+
+    |sample                | t1                | ENCRYPTION="Y"               |
+    +----------------------+-------------------+------------------------------+
+
+A ``flag`` field in the ``INFORMATION_SCHEMA.INNODB_TABLESPACES`` has the bit
+number 13 set if the tablespace is encrypted. This bit can be checked with the
+``flag & 8192`` expression with the following method:
+
+.. code-block:: mysql
+
+    SELECT space, name, flag, (flag & 8192) != 0 AS encrypted FROM
+    INFORMATION_SCHEMA.INNODB_TABLESPACES WHERE name in ('foo', 'test/t2', 'bar',
+    'noencrypt');
+
+      +-------+-----------+-------+-----------+
+      | space | name      | flag  | encrypted |
+      +-------+-----------+-------+-----------+
+      |    29 | foo       | 10240 |      8192 |
+      |    30 | test/t2   |  8225 |      8192 |
+      |    31 | bar       | 10240 |      8192 |
+      |    32 | noencrypt |  2048 |         0 |
+      +-------+-----------+-------+-----------+
+      4 rows in set (0.01 sec)
+
+:Availability: This feature is **Experimental**.
+
+Encrypted table metadata is contained in the
+``INFORMATION_SCHEMA.INNODB_TABLESPACES_ENCRYPTION`` table.
+
+You must have the `Process
+<https://dev.mysql.com/doc/refman/8.0/en/privileges-provided.html#priv_process>`__
+privilege to view the table information.
+
+.. note::
+
+    This table is **Experimental** and may change in future releases.
+
+.. code-block:: MySQL
+
+  >desc INNODB_TABLESPACES_ENCRYPTION:
+
+  +-----------------------------+--------------------+-----+----+--------+------+
+  | Field                       | Type               | Null| Key| Default| Extra|
+  +-----------------------------+--------------------+-----+----+--------+------+
+  | SPACE                       | int(11) unsigned   | NO  |    |        |      |
+  | NAME                        | varchar(655)       | YES |    |        |      |
+  | ENCRYPTION_SCHEME           | int(11) unsigned   | NO  |    |        |      |
+  | KEYSERVER_REQUESTS          | int(11) unsigned   | NO  |    |        |      |
+  | MIN_KEY_VERSION             | int(11) unsigned   | NO  |    |        |      |
+  | CURRENT_KEY_VERSION         | int(11) unsigned   | NO  |    |        |      |
+  | KEY_ROTATION_PAGE_NUMBER    | bigint(21) unsigned| YES |    |        |      |
+  | KEY_ROTATION_MAX_PAGE_NUMBER| bigint(21) unsigned| YES |    |        |      |
+  | CURRENT_KEY_ID              | int(11) unsigned   | NO  |    |        |      |
+  | ROTATING_OR_FLUSHING        | int(1) unsigned    | NO  |    |        |      |
+  +-----------------------------+--------------------+-----+----+--------+------+
+
+To verify if the binary log encryption option is enabled, run the following
+statement:
+
+.. code-block:: MySQL
+
+    mysql>SHOW BINARY LOGS;
```

Comment:
this 8.0 only feature - the Encrypted column in SHOW BINARY LOGS output.

Reply:
Removed information and the example for the SHOW BINARY LOGS.